src/ontology/nbo-edit.owl

```diff
@@ -9938,6 +9944,27 @@ AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0050002 "behavior_ontology"
 AnnotationAssertion(rdfs:label obo:NBO_0050002 "circling behavior")
 SubClassOf(obo:NBO_0050002 obo:NBO_0000013)
 
+# Class: obo:NBO_0070001 (migration)
+
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0070001 "Rhythmic behavior that involves cue-initiated, persistent, direct, focused movement between habitats.")
+AnnotationAssertion(terms:contributor obo:NBO_0070001 <https://orcid.org/0000-0002-4277-2151>)
+AnnotationAssertion(terms:date obo:NBO_0070001 "2023-04-22T14:17:43Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasRelatedSynonym obo:NBO_0070001 "Movement ecology")
+AnnotationAssertion(oboInOwl:source obo:NBO_0070001 "Dingle H. Migration: the biology of life on the move. Oxford University Press, USA; 2014.")
+AnnotationAssertion(rdfs:label obo:NBO_0070001 "migration")
+SubClassOf(obo:NBO_0070001 obo:NBO_0000008)
+
+# Class: obo:NBO_0070002 (diel vertical migration)
+
+AnnotationAssertion(obo:IAO_0000115 obo:NBO_0070002 "Migration that involves regular movement up and down in the water column.")
+AnnotationAssertion(dc:contributor obo:NBO_0070002 <http://orcid.org/0000-0002-2908-3327>)
+AnnotationAssertion(terms:contributor obo:NBO_0070002 <https://orcid.org/0000-0002-4277-2151>)
+AnnotationAssertion(terms:date obo:NBO_0070002 "2023-04-22T14:41:53Z"^^xsd:dateTime)
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:NBO_0070002 "behavior_ontology")
+AnnotationAssertion(oboInOwl:source obo:NBO_0070002 "Bandara K, Varpe Ø, Wijewardene L, Tverberg V, Eiane K. Two hundred years of zooplankton vertical migration research. Biological Reviews. 2021 Aug;96(4):1547-89.")
```

Comment:
We usually add references as ISBN, ISBN13 or DOI directly to the definition.

<img width="824" alt="image" src="https://user-images.githubusercontent.com/7070631/234201914-944567f1-570f-4118-8ff7-7d7ea3e72512.png">

Reply:
1. I've added the DOI and will make changes to the other PRs.
2. I have had 2 weeks or more suggested changes and conversations for each issue before submitting them as PRs, so I'm not expecting you to review content, nor have I put them forward without the opportunity for comment, but good for you to double-check.